README.md

```diff
@@ -4,8 +4,6 @@
 [![Version](https://img.shields.io/cocoapods/v/Hyperspace.svg?style=flat)](http://cocoapods.org/pods/Hyperspace)
 [![License](https://img.shields.io/cocoapods/l/Hyperspace.svg?style=flat)](http://cocoapods.org/pods/Hyperspace)
 [![Platform](https://img.shields.io/cocoapods/p/Hyperspace.svg?style=flat)](http://cocoapods.org/pods/Hyperspace)
-[![codecov](https://codecov.io/gh/BottleRocketStudios/iOS-Hyperspace/branch/master/graph/badge.svg)](https://codecov.io/gh/BottleRocketStudios/iOS-Hyperspace)
```

Comment:
Same again - I think it's a product of the merge. But we'll want both of these badges to remain present in the Readme

Reply:
I'm getting my branches mixed up. Ignore me.

Reply:
Added both codecov & codebeat from db315ed don't know why they didn't merge in properly.